src/olympia/devhub/templates/devhub/addons/listing/delete_form.html

```diff
@@ -23,7 +23,7 @@
   {% endif %}
   <p>
     <label>
-      {{ _('Enter the following text confirm your decision: {slug}')|fe(slug=addon.slug) }}
+      {{ _('Enter the following text to confirm your decision: {slug}')|fe(slug=addon.slug) }}
```

Comment:
I think this was included accidentally, can you remove it from this pull request, please?

It looks like you're working on the master branch. I'd suggest you work on a new branch for each change.